.travis.yml

```diff
@@ -168,6 +168,8 @@ jobs:
             condition: $DEPLOY_DOCKER != 0 && $TRAVIS_TAG =~ ^v?[[:digit:]]+\.[[:digit:]]+\.[[:digit:]]$ && ($TRAVIS_REPO_SLUG = 'codedellemc/rexray' || $IGNORE_REPO_SLUG_CONDITION = true)
 
     - <<: *docker-stage
+      env: DRIVER=cinder
```

Comment:
Hi @codenrhoden,

Not to nitpick, but should this be the same commit as Azure UMD? I don't mind it being the same PR, but maybe rebase it into a separate commit specific to Cinder?

Reply:
This build stages are in alphabetic order, and azure comes before Cinder. So this is really just adding Azure to the build stages, not anything to do with Cinder.

Reply:
Hi @codenrhoden,

Ah, sorry. Mea culpa. Sorry to pick some bad nits.